### PR TITLE
Spanish (2 files) + Korean (5 files) translation

### PR DESCRIPTION
**Pull request by reuptaken**
There's an issue with links in one doc (Korean version) which will be fixed in next commit.

### CODE REVIEW

File: faqs/ko/emergency-shutdown.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+# 비상정지 FAQ
+
+## 비상정지란 무엇인가요?
+
+비상정지란 Dai의 주요 보안 기능 중 하나이며 이전에는 포괄적 합의 (Global Settlement)라고 불렸습니다. 이 중요한 보안 기능으로 시스템을 정지시킬 수 있으며 [Dai](https://github.com/makerdao/community/blob/master/faqs/dai.md)와 CDP 소유자들이 담보를 회수할 수 있도록 합니다.
```
Comment (ghettodev) [inline faqs/ko/emergency-shutdown.md]: links should be relative

File: faqs/ko/emergency-shutdown.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+2. 담보 유치권에 대한 처리가 진행됩니다:
+   비상정지가 발동된 후, 모든 CDP 소유자들의 담보 유치권을 처리하기위한 시간이 필요합니다. 이 과정이 끝난 후 모든 CDP 소유자들은 본인의 CDP를 통해 특정액의 ETH를 받을 수 있습니다. 반면, Dai 소유자들은 담보 유치권을 즉시 행사할 수 있습니다.
+
+3. Dai와 CDP 소유자들이 담보를 받아갑니다:
+	모든 Dai와 CDP 소유자들은 본인의 Dai와 CDP들의 가치를 계산하여, 그 가치에 해당하는 ETH로 바로 교환할 수 있습니다.
+
+## 누가 비상정지를 발동할 수 있나요?
+
+단일 담보 기반 Dai에서 비상정지를 발동시킬 수 있는 유일한 방법은 MKR 투표자들이 [실행 투표](faqs/governance.md#is-there-more-than-one-type-of-vote)를 통과시키는 방법입니다.
```
Comment (ghettodev) [inline faqs/ko/emergency-shutdown.md]: this relative link should be in the same dir. 

`[실행 투표](governance.md#is-there-more-than-one-type-of-vote)`

File: faqs/ko/glossary.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
+# 용어 해설집
+
+## A
+
+## B
+
+* **바이트(Bite)**: CDP의 청산을 일으키는 행위
```
Comment (ghettodev) [inline faqs/ko/glossary.md]: These subheadings need to follow the same syntax as the English version so we can link directly to the definitions:

```
# Glossary of Terms

## A

## B

##### Bite

- the action of triggering a CDPs liquidation
```

File: faqs/es/glossary.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+# Glosario de Términos
+
+## A
+
+* **Apagado de Emergencia**: una prestación de seguridad que permite al sistema detenerse y ofrecer el colateral subyacente como amortización
```
[the remark this answers is not in the document]
Comment (reuptaken) [reply faqs/es/glossary.md]: I'll let editors know. They used version of this document with different formatting.

**Comment by reuptaken** [comment]
I'm closing it and will open another one.

**Pull request closed without merging**